src/fielddef.ts

```diff
@@ -182,12 +182,13 @@ export interface ScaleFieldDef<F> extends FieldDef<F> {
 
   /**
    * Sort order for the encoded field.
-   * Supported `sort` values include `"ascending"`, `"descending"` and `null` (no sorting).
+   * Supported `sort` values include `"ascending"`, `"descending"`, `array specifying the preferred order of values` and `null` (no sorting).
```

Comment:
The `sort` property can be `"ascending"`, `"descending"`, `null`, or an array specifying the preferred order of values.

(Remove back ticks for the array case too since you have currently have inside backticks is not the exact string you gonna put in the JSON.)